app/models/sitemap_indexer.rb

```diff
@@ -10,21 +10,20 @@ def index
     sitemap = Sitemaps.discover(domain)
     Rails.logger.info "[Searchgov SitemapIndexer] #{log_info.merge(sitemap_entries_found: sitemap.entries.count).to_json}"
     sitemap.entries.each{ |entry| process_entry(entry) }
-    SearchgovUrl.fetch_new
```

Comment:
This `fetch_new` business will be completely rejiggered when I build a sane queuing system. I'm removing it from the bulk indexing task and the sitemap indexer because a) it makes those processes run much longer than they should, and b) it runs the risk of multiple processes fetching URLs for the same domains, which could result in  URLs for within a domain being fetched too quickly, and getting us blocked.